browser/brave_app_controller_mac.mm

```diff
@@ -15,21 +15,24 @@
 
 @implementation BraveAppController
 
+@synthesize copyMenuItem = _copyMenuItem;
+@synthesize copyCleanLinkMenuItem = _copyCleanLinkMenuItem;
+
```

Comment:
Shouldn't property synthesis be done automatically by the compiler? Are we missing a flag possibly?

Reply:
cc @sangwoo108